test/widgets/subscription_list_test.dart

```diff
@@ -190,4 +190,41 @@ void main() {
     check(tester.widget<UnreadCountBadge>(find.byType(UnreadCountBadge)).backgroundColor)
       .equals(swatch);
   });
+
+  testWidgets('muted streams are displayed as faded', (tester) async {
+    final stream = eg.stream(name: 'Channel 1');
+    final stream2 = eg.stream(streamId: 2, name: 'Channel 2');
+    final unreadMsgs = eg.unreadMsgs(streams: [
+      UnreadStreamSnapshot(streamId: stream.streamId, topic: 'a', unreadMessageIds: [1, 2]),
+      UnreadStreamSnapshot(streamId: stream2.streamId, topic: 'b', unreadMessageIds: [3]),
+    ]);
+    await setupStreamListPage(tester,
+      subscriptions: [
+        eg.subscription(stream, isMuted: true),
+        eg.subscription(stream2, isMuted: false)
+      ],
+      userTopics: [
+        UserTopicItem(
+          streamId: stream.streamId,
+          topicName: 'a',
+          lastUpdated: 1234567890,
+          visibilityPolicy: UserTopicVisibilityPolicy.unmuted,
+        ),
+        UserTopicItem(
+          streamId: stream2.streamId,
+          topicName: 'b',
+          lastUpdated: 1234567890,
+          visibilityPolicy: UserTopicVisibilityPolicy.unmuted,
+        ),
+      ],
+      unreadMsgs: unreadMsgs);
+
+    final channel1Finder = find.text('Channel 1');
+    final channel1Opacity = tester.widget<Opacity>(find.ancestor(of: channel1Finder, matching: find.byType(Opacity))).opacity;
+    check(channel1Opacity).isLessThan(1.0);
```

Comment:
```suggestion
    check(stream1Opacity).equals(0.55);
```
Let's use the exact opacity value used in the code to make sure this test fails if there is an unintentional change to the opacity.